### DISCUSSION TITLE
IP Pools Settings  Behind NAT

### DISCUSSION THREAD

**Discussion by onurbektas** [postalserver/postal#2189] [Help with using Postal]
I am trying to  configure ip pool for servers behind NAT.  Each server has its private IP address. Private IPs is translated to  public IPs  via 1:1 NAT to connect the Internet.

IP pool documentation clearly state that the IP added  to IP pool should be assigned on servers. But for our case, public IP addresses are not assigned on servers. I tried to add private IP address to ip pool list but it does not work. Is there any way to make ip pools work behind NAT'd ip addresses ?

**Comment by willpower232** [comment]
> I tried to add private IP address to ip pool list but it does not work

Can you definitely see the private addresses with `ip a` on the server? What you describe should be fine but Postal will need to be able to see some IP addresses directly to use them, IP Pools or not.

**Comment by onurbektas** [comment]
Yes, I can see that private IP address is assigned on server. I solved the problem by using real IPs instead of private IP addresses and changing the ip pools accordingly.
